src/api/create-vis.js

```diff
@@ -34,6 +34,7 @@ var createVis = function (el, vizjson, options) {
     apiKey: options.apiKey,
     authToken: vizjson.auth_tokens && vizjson.auth_tokens.length && vizjson.auth_tokens[0],
     showLegends: options.legends === true || vizjson.legends === true,
+    showLogo: options.logo === undefined || options.logo === true || vizjson.logo === true,
```

Comment:
We could also add a default value for showLogo [here](https://github.com/CartoDB/cartodb.js/blob/66740c833adfeb8729e0bfe6966f953a17e5e27c/src/api/create-vis.js#L7). and remove the `options.logo === undefined`...